bundles/org.openhab.binding.semsportal/README.md

```diff
@@ -0,0 +1,45 @@
+# SEMSPortal Binding
+
+This binding can help you include statistics of your SEMS / GoodWe solar panel insallation into openHAB. It is a readonly connection that maps collected parameters to openhab channels. It provides current, day, month and total yields, as well as some income statistics if you have configured these in the SEMS portal. It requires a power station that is connected through the internet to the SEMS portal.
```

Comment:
```suggestion
This binding can help you include statistics of your SEMS / GoodWe solar panel insallation into openHAB. 
It is a read only connection that maps collected parameters to openHAB channels. 
It provides current, day, month and total yields, as well as some income statistics if you have configured these in the SEMS portal. 
It requires a power station that is connected through the internet to the SEMS portal.
```

Use openHAB not openhab, note uppercase HAB.
Also please place each sentence on a new line as it makes future changes cleaner when people do a PR.
Can you go over the entire file and make sure each sentence is a newline please?

Reply:
Done.